content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -185,6 +185,13 @@ If you have a continued need for a Russian localization of the Cumulocity IoT UI
 
 #### Implemented
 
+
+##### Changes in Gainsight tracking
+
+In efforts to get more insights from our users, we enable tracking for every user that has agreed or rejected the cookie and policy banner. In case of rejection, the user will be tracked without any personal identifiable information (pii) like e-mail, name or IP address. Not tracked are still users that haven't been shown a cookie/policy banner or users that are using dedicated instances without any Gainsight API key set.
```

Comment:
```suggestion
In efforts to get more insights into the user behavior, we enable tracking for every user that has agreed or rejected the cookie policy. In case of rejection, the user will be tracked without any personal identifiable information (pii) like email address, name or IP address. We still do not track users that haven't been shown a cookie/policy banner or users that are using dedicated instances without any Gainsight API key set.
```